classes/OccurrenceSearchSupport.php

```diff
@@ -84,7 +85,7 @@ public function getFullCollectionList($catIdStr = '', $limitByImages = false){
 		return $retArr;
 	}
 
-	public function outputFullCollArr($collGrpArr, $targetCatID = '', $displayIcons = true, $displaySearchButtons = true){
+	public function outputFullCollArr($collGrpArr, $targetCatID = '', $displayIcons = true, $displaySearchButtons = true, $collTypeLabel = '', $uniqGrouping=''){
```

Comment:
New arguments added to ensure uniqueness with some of the labels and ids, per feedback from TotalValidatorPro.